pkg/core/blockchain.go

```diff
@@ -2480,6 +2491,11 @@ func (bc *Blockchain) verifyTxAttributes(d *dao.Simple, tx *transaction.Transact
 			if !tx.HasSigner(bc.contracts.Notary.Hash) {
 				return fmt.Errorf("%w: NotaryAssisted attribute was found, but transaction is not signed by the Notary native contract", ErrInvalidAttribute)
 			}
+		case transaction.RefundableSystemFeeT:
+			state := bc.GetContractState(tx.Sender())
+			if state != nil {
+				return fmt.Errorf("%w: RefundableSystemFee attribute was found, but transaction sender is contract", ErrInvalidAttribute)
+			}
```

Comment:
Just a question, how about the destroyed contracts? `bc.GetContractState(tx.Sender())` returns nil for them.

Reply:
I think deleted contract can't pass witness check